### DISCUSSION TITLE
Deploy status website to Netlify

### DISCUSSION THREAD

**Discussion by herrbuerger** [upptime/upptime#766] [Q&A] [answered]
Hello,

I have a question about deploying the status website to Netlify:
Here's what I did so far:

- setup Upptime
- setup a site on Netlify and have it deploy from the `gh_pages` branch

Up to this point everything is working fine and the status page is working properly. However I need to add a `netlify.toml` file to the `gh_pages` branch to configure some things. This file is unfortunately overwritten by the `upptime-bot` everytime he makes a commit. 

Is there any way to get this file into the branch and not have it removed everytime?

For some more context why I'm using Netlify. I have a private repo and it seems like I found an easy solution for proxying requests to Github. Happy to share that with the community, but without the `netlify.toml` file it won't work.

Best,
Christian

**Comment by AnandChowdhary** [comment, marked as answer]
I suggest you add `netlify.toml` in the `assets` directory, and then it will be added in the root directory of the gh-pages branch
